### PR TITLE
Requires OAuthToken set to use GitHub3Plugin features

### LINKED ISSUE #6757: [NBug] Object reference not set to an instance of an object.
## Current behaviour
No pull request window or redirect to browser happened. See an exception window instead

## Expected behaviour
No exception window, create pull request dialog is opened

## Steps to reproduce
1. Do Not authenticate on Github inside Git Extensions
2. Press GitHub=>Create Pull Request ... menu item for existing branch
3. Exception occurred with an option to send bug

## Error Details
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Git.hub.Repository.GetBranches()
   at GitHub3.GitHubRepo.GetBranches()
   at GitUI.CommandsDialogs.RepoHosting.CreatePullRequestForm.<>c__DisplayClass22_0.<<PopulateBranchesComboAndEnableCreateButton>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Environment
- Git Extensions 3.1.1.6049
- Build 2f872105b4eb42421d9fdafd68e90a9e27d20d96
- Git 2.20.0.windows.1
- Microsoft Windows NT 10.0.16299.0
- .NET Framework 4.7.3394.0
- DPI 96dpi (no scaling)

### PR DESCRIPTION
**Pull request by pmiossec**
That will open the web browser window to generate the token if needed.

+ fix a duplicate string

Fixes #6757 now that github api will return the good data instead of a 403

## Test methodology 

- manual

### CODE REVIEW

File: GitUI/GitUICommands.cs [hunk 1]
```diff
@@ -1326,7 +1326,7 @@ private void WrapRepoHostingCall(string name, IRepositoryHostPlugin gitHoster, A
 
         public void StartCloneForkFromHoster(IWin32Window owner, IRepositoryHostPlugin gitHoster, EventHandler<GitModuleEventArgs> gitModuleChanged)
         {
-            WrapRepoHostingCall("View pull requests", gitHoster, gh =>
+            WrapRepoHostingCall("Fork or clone a repository", gitHoster, gh =>
```
Comment (RussKie) [inline GitUI/GitUICommands.cs]: We need to translate this

Comment (pmiossec) [reply GitUI/GitUICommands.cs]: done.

**Comment by RussKie** [comment]
Could you please also update translations.

I just pushed into master an updated script (e7688f6fd82614979d964cf69233b951ad521970).
You could either rebase on top of the latest master or just cherry-pick it, and run it:

```powershell
PS> pushd .\Build\transifex
PS> .\UpdateLocalEnglishTranslations.ps1 -Configuration Debug
```

**Comment by pmiossec** [comment]
>Could you please also update translations.

I don't know why but the strings doesn't end up in the xlf files. I have other strings updated but not these ones 😢 
I don't know what is the rule to find `TranslationString`...

**Comment by pmiossec** [comment]
>
> Dont really see why it works now and there is the history bug.
>

The main reason of the exception is that the github api returned a 403
error because the api is queried without token. So some calls failed,
throwing exceptions that were swallowed.

 All that end up with the 'Owner' property not set, so the value is null,
which is used for a last call and causing a NRE.

The change made here is that for all the feature provided by the github
plug-in, before opening the form, we check if the token is filled. If, not,
the web page to generate it is open.

All the code was already there, just missing the good implementation of
'IsConfigurationOK'...

This fix prevents the NRE to happen in the nominal scenario but looking at
the code, all api errors are not well handled and would require a lot of
work.

>

**Comment by RussKie** [comment]
> I don't know why but the strings doesn't end up in the xlf files

I'll have a look

**Comment by RussKie** [comment]
> I don't know what is the rule to find `TranslationString`...

Translatable strings must be defined in a class that implements `ITranslate`. Have a look at how `ResourceManager.Strings` is implemented.

**Pull request merged**